src/scene/container/RenderContainer.ts

```diff
@@ -19,8 +19,21 @@ export interface RenderContainerOptions extends ContainerOptions
     render?: RenderFunction;
     /** how to know if the custom render logic contains a point or not, used for interaction */
     containsPoint?: (point: Point) => boolean;
-    /** how to add the bounds of this object when measuring */
+    /** @deprecated how to add the bounds of this object when measuring */
     addBounds?: (bounds: BoundsData) => void;
```

Comment:
```suggestion
    /**
      * how to add the bounds of this object when measuring 
      * @deprecated since 8.3.0
      * @see ViewContainer#updateBounds
      */
    addBounds?: (bounds: BoundsData) => void;
```